tests/scripts/arm64_post_build.py

```diff
@@ -280,10 +323,10 @@ def main(args):
            build_type, 
            scenario, 
            core_root, 
-           g_test_url]
+           test_location]
 
    log(" ".join(args))
-   proc = subprocess.Popen(args, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+   proc = subprocess.Popen(args)
    std_out, std_err = proc.communicate()
```

Comment:
The correct implementation requires using Popen and stdout. We could technically use check_output, but I do not think the exception being thrown here gives us any advantage.